Formula/homeshick.rb

```diff
@@ -0,0 +1,68 @@
+class Homeshick < Formula
+  desc "Git dotfiles synchronizer written in bash"
+  homepage "https://github.com/andsens/homeshick"
+  url "https://github.com/andsens/homeshick/archive/1.0.0.tar.gz"
+  sha256 "8bd3c46f1cfd68d82d97fa72a68a07c966092c77f276d1335cb390b2ec6062bf"
+  head "https://github.com/andsens/homeshick.git"
+
+  option "with-fish", "Build fish bindings"
+  option "with-csh", "Build csh bindings"
+  option "without-bash", "Build without bash bindings"
+
+  def install
+    inreplace "bin/homeshick", /^homeshick=.*/, "homeshick=#{opt_prefix}"
+
+    prefix.install "lib"
+    prefix.install "homeshick.sh"
+    prefix.install "homeshick.fish" if build.with? "fish"
+    bin.install "bin/homeshick"
+    bin.install "bin/homeshick.csh" if build.with? "csh"
+    zsh_completion.install "completions/_homeshick"
+    bash_completion.install "completions/homeshick-completion.bash"
+    if build.head? && (build.with? "fish")
+      fish_completion.install "completions/homeshick.fish"
+    end
+  end
+
+  def caveats
+    s = <<-EOS.undent
+      `export HOMESHICK_DIR=#{opt_prefix}`
```

Comment:
To enable the subcommand `homeshick cd <CASTLE>` yes as it required the shell wrappers which in turn use $HOMESHICK_DIR to execute the binary homeshick. Changeing the shell wrapper would take a couple more `inreplace` commands.
The only other option I'd see would be to tell users in the caveat the content of those wrappers (which is a small shell function). @andsens @DomT4 @MikeMcQuaid what would be your take on something like this:

instead of those shell wrappers (export & sourceing) we could print something like:
```
To enable `homeshick dir <CASTLE>` command append

function homeshick() {
	if [ "$1" = "cd" ] && [ -n "$2" ]; then
		cd "$HOME/.homesick/repos/$2"
	else
		"#{opt_prefix}/bin/homeshick" "$@"
	fi
}

to your $HOME/.bashrc
```